### DISCUSSION TITLE
Vote to add Serpent OS to Distrowatch database

### DISCUSSION THREAD

**Discussion by NomadicCore** [AerynOS/discussions#1] [General]
Serpent OS is in the early stages but with the pre-alpha being very quickly iterated on, I think it would be good to get it upvoted on Distrowatch and formally added to the database. That won't happen overnight of course.

You can do this by following this link and clicking the recommend button next to Serpent OS.

https://distrowatch.com/dwres.php?resource=links#new

**Comment by ikeycode** [comment]
For some context DW looked at Serpent OS a very long time ago in our snekvalidator iso stages and I let them know the project wasn't anywhere near "ready".

I believe they've been respecting that wish thus I respect them even more for that. That said we're in a very different stage now, so I'll shoot over an email explaining the change if necessary.

**Comment by NomadicCore** [comment]
Ah okay, I didn't know they were holding off, definately props to them for respecting that.

Given what you've said, I agree that the project is in a very difference space now. I guess it comes down to when you want to get more visibility and coverage on the project? I'd suggest maybe holding off asking them to take a look until after the next cosmic spin so the sudo issue is sorted for example. To that end, would it make sense to wait until the project gets to an alpha tag rather than pre-alpha?

**Reply by ikeycode** [reply]
Sure, lets get an alpha tag out!

**Comment by chrisdebian** [comment]
And, to [Repology](https://repology.org/docs/requirements)?

Cheers,

Chris.

**Reply by NomadicCore** [reply]
It's already there

https://repology.org/repository/serpentos

**Comment by ikeycode** [comment]
For the record I reached out to Distrowatch yesterday to indicate my belief the project is now in a more tenable position

**Comment by chrisdebian** [comment]
Thanks, Ikey; good call.

Chris
